### ISSUE TITLE
allow Path annotation on interface for resources

### ISSUE THREAD

**Issue by brackxm** [xvik/dropwizard-guicey#10]
Allow the Path annotation on an interface for resources.

**Comment by xvik** [comment]
Please describe the complete use case

**Comment by brackxm** [comment]
The use case adding a resource class (e.g. via auto scanning) that does not directly have a @Path annotation. Instead the annotation is on in interface that the class implements.
The matches check in ResourceInstaller via FeatureUtils.hasAnnotation() does not check interfaces, but only (super) classes.

```
public class SampleResource implements Sample {
}

@Path("/sample")
public interface Sample {
}
```

**Comment by xvik** [comment]
Agree, this makes sense.

**Comment by xvik** [comment]
Released with version [3.3.0](https://github.com/xvik/dropwizard-guicey/releases/tag/3.3.0)
